src/server/services/buurt/buurt.ts

```diff
@@ -260,3 +272,81 @@ export async function loadFeatureDetail(
 
   return response;
 }
+
```

Comment:
Kun je deze wior functionaliteit in een aparte file zetten? Wat mij betreft mag die transform functie uit datasets.ts er ook in.